build.cake

```diff
@@ -1,145 +1,93 @@
+        CleanDirectory(pathHelloWorldIntegrationTest + "CreateInstancesReactiveApi/CreateInstances/bin");
+        CleanDirectory(pathHelloWorldIntegrationTest + "CreateInstancesReactiveApi/CreateInstances/obj");
     });
 
 Task("Build")
     .Does(() =>
     {
-        NuGetRestore("ReactiveXComponent.sln", new NuGetRestoreSettings { NoCache = true });
-        BuildSolution(@"./ReactiveXComponent.sln", buildConfiguration, setAssemblyVersion, version);
+        DotNetCoreRestore("ReactiveXComponent.sln");
+        DotNetCoreBuild(
+        "ReactiveXComponent.sln",
+        new DotNetCoreBuildSettings {
+            Configuration = buildConfiguration,
+        });
     });
 
 Task("Test")
     .Does(() =>
     {
-        var testAssembliesPatterns = new string[]
+        var settings = new DotNetCoreTestSettings
         {
-            "./ReactiveXComponentTest/bin/" + buildConfiguration + "/ReactiveXComponentTest.dll"
+            Configuration = buildConfiguration
         };
-        
-        var testAssemblies = GetFiles(testAssembliesPatterns);
-        var nunitSettings = new NUnit3Settings(){ Results = new[] { new NUnit3Result { FileName = "TestResults.xml" } } };
-        NUnit3(testAssemblies, nunitSettings);
-    });
 
-Task("Merge")
-    .Does(() =>
-    {
-        EnsureDirectoryExists("packaging");
-
-        var filesToMerge = GetFiles("./ReactiveXComponent/bin/"+ buildConfiguration + "/*.dll");
-
-        var ilRepackSettings = new ILRepackSettings { Parallel = true, Internalize = true };
-
-        ILRepack(
-            "./packaging/ReactiveXComponent.dll",
-            "./ReactiveXComponent/bin/"+ buildConfiguration + "/ReactiveXComponent.dll",
-            filesToMerge,
-            ilRepackSettings
-		);
-
-        var pdbFiles = GetFiles("./ReactiveXComponent/bin/"+ buildConfiguration + "/ReactiveXComponent.pdb");
-        CopyFiles(pdbFiles, "./packaging");
+        var projectFiles = GetFiles("./**/*Test.csproj");
+        foreach(var file in projectFiles)
+        {
+            DotNetCoreTest(file.FullPath, settings);
+        }
     });
 
 Task("CreatePackage")
-    .IsDependentOn("Merge")
     .Does(() =>
     {
-        EnsureDirectoryExists("nuget");
-
-        var formattedNugetVersion = FormatNugetVersion(version);
-
-        var filesToPackPatterns = new string[]
-        {
-            "./packaging/*.dll",
-            "./packaging/*.pdb"
-        };
-
-        var filesToPack = GetFiles(filesToPackPatterns);
-
-        var nuSpecContents = filesToPack.Select(file => new NuSpecContent {Source = file.FullPath, Target = @"lib\net451"}).ToList();
-
-        var nugetPackSettings = new NuGetPackSettings()
-        { 
-            OutputDirectory = @"./nuget",
-            Files = nuSpecContents,
-            Version = formattedNugetVersion,
-            IncludeReferencedProjects = true
-        };
-
-        NuGetPack("ReactiveXComponent.Net.nuspec", nugetPackSettings);
+        DotNetCorePack(
```

Comment:
Did you check the nuget package created by this cake task ?
We do not use ReactiveXComponent.Net.nuspec anymore. Is it a mistake ?

Reply:
Nuspec is the old way to create nugets. It's integrated in the csproj with dotnetcore/standard

Reply:
So @fle-bach just remove the old .nuspec file from the repository

Reply:
yes i do it when my error was corrected :)

Reply:
should i delete the two .nuspec this in main directory and this in ReactiveXComponent directory?

Reply:
Yes both

Reply:
i do it, i will add in .csproj